OWML.ModHelper.Input/OWML.ModHelper.Input.csproj

```diff
@@ -81,7 +81,6 @@
     <Compile Include="ModInputTextures.cs" />
     <Compile Include="ModInputLibrary.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="ModInputLibrary.cs" />
```

Comment:
Not sure what this means, it's still there... 🤷

Reply:
ah, that's a random duplicate. I kinda forgot to remove it